src/components/tile/tile.stories.tsx

```diff
@@ -161,6 +162,107 @@ export const WithTileFooter: ComponentStory<typeof Tile> = () => (
   </Box>
 );
 
+export const WithTileHeader: ComponentStory<typeof Tile> = () => (
```

Comment:
**issue:** It looks like this story has a visual defect. It looks like the squared corners are sticky out of the rounded corners.

![Screenshot 2024-02-09 at 11 05 27](https://github.com/Sage/carbon/assets/56251247/70a60c1d-3e56-474a-96c2-cfb00990cb5a)